tls-codec/tls_codec/benches/tls_vec.rs

```diff
@@ -0,0 +1,97 @@
+#[macro_use]
+extern crate criterion;
```

Comment:
Minor nit, but as a 2018 edition crate the relevant macros can be imported using the module syntax:

```suggestion
use criterion::{criterion_group, criterion_main};
```

Likewise the `extern crate tls_codec` below is unnecessary.